### ISSUE TITLE
False positive for MultilineTernaryOperator

### ISSUE THREAD

**Issue by AlexWayfer** [rubocop-hq/rubocop#7466]
Hello!

I agree with `MultilineTernaryOperator`. But not for cases when a ternary operator is on single line, but on the second line as argument.

--------

## Expected behavior

No offenses.

## Actual behavior

Offense from `MultilineTernaryOperator`.

## Steps to reproduce the problem

```ruby
array <<
  foo ? bar : baz # can't place on the previous line due to line length
```

## RuboCop version

```
> rubocop -V
0.75.1 (using Parser 2.6.5.0, running on ruby 2.6.4 x86_64-linux)
```

**Comment by buehmann** [comment]
Hi! It seems like you are mistaken about the predence of `<<` and `? :`. Your code is equivalent to
```ruby
(array <<
  foo) ? bar : baz # can't place on the previous line due to line length
```
not to
```ruby
array <<
  (foo ? bar : baz) # can't place on the previous line due to line length
```

**Comment by AlexWayfer** [comment]
Oh… 

Thank you, you're right.

Example: https://repl.it/@AlexWayfer/ThistleFairInitializationfile
